docs/guides/mysql/concepts/mysqldatabase/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: MySQLDatabase
+menu:
+  docs_{{ .version }}:
+    identifier: mysqldatabase-concepts
+    name: MySQLDatabase
+    parent: guides-mysql-concepts
+    weight: 30
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# MySQLDatabase
+
```

Comment:
add this : 
What is MySQLDatabase ? 
MySQLDatabase is a Kubernetes Custom Resource Definitions (CRD). It provides  a declarative way of implementing multitenancy inside KubeDB provisioned MySQL server. You need to describe the target database, desired database configuration, the vault server reference for managing the user in a MySQLDatabase object, and the KubeDB Schema Manager operator will create Kubernetes objects in the desired state for you.